packages/mantle/src/components/tabs/tabs.tsx

```diff
@@ -169,21 +218,41 @@ const Trigger = forwardRef<
 		},
 		ref,
 	) => {
-		const ctx = useContext(TabsStateContext);
+		const { orientation, appearance } = useContext(TabsStateContext);
 		const disabled = parseBooleanish(_ariaDisabled ?? _disabled);
 
+		const variants = cva("", {
```

Comment:
it's small, but let's hoist these `cva` calls out to the module level

this one can be `triggerVariants`

we should also take the existing `className` below (around L249-254) and plop it into the first arg of `cva` here